### ISSUE TITLE
Build fails on src/qtwebkit/viewwindowmgr.cpp : incomplete type ‘class QTabBar’

### ISSUE THREAD

**Issue by u-235** [u-235/kchmviewer#5]
From [unixforum](https://unixforum.org/viewtopic.php?p=1349386#p1349386)

--------

- slackware 14.2 x64
- qt5-5.12.8-x86_64-sl14.2
- qt5_webkit-5.9.1-x86_64-sl14.2_alien

```
[ 16%] Building CXX object src/CMakeFiles/kchmviewer.dir/qtwebkit/viewwindowmgr.cpp.o
/tmp/sun/x/kchmviewer-8.1/kchmviewer-8.1-rc/src/qtwebkit/viewwindowmgr.cpp: In member function ‘virtual void ViewWindowTabWidget::mouseReleaseEvent(QMouseEvent*)’:
/tmp/sun/x/kchmviewer-8.1/kchmviewer-8.1-rc/src/qtwebkit/viewwindowmgr.cpp:41:23: error: invalid use of incomplete type ‘class QTabBar’
     int tab = tabBar()->tabAt( event->pos() );
                       ^
```

**Comment by arzam16** [comment]
I've got the same error.

- Linux Mint 20.2
- qt5-default:amd64 5.12.8+dfsg-0ubuntu1
- libqt5webkit5-dev:amd64 5.212.0~alpha4-1ubuntu2

```
LANGUAGE=en_US.UTF-8 cmake --build . --config Release -j32
[ 13%] Built target translations
[ 28%] Built target ebook
[ 29%] Building CXX object src/CMakeFiles/kchmviewer.dir/qtwebkit/viewwindowmgr.cpp.o
/tmp/tmp.f9IFZjFN3X/kchmviewer/src/qtwebkit/viewwindowmgr.cpp: In member function ‘virtual void ViewWindowTabWidget::mouseReleaseEvent(QMouseEvent*)’:
/tmp/tmp.f9IFZjFN3X/kchmviewer/src/qtwebkit/viewwindowmgr.cpp:41:23: error: invalid use of incomplete type ‘class QTabBar’
   41 |     int tab = tabBar()->tabAt( event->pos() );
      |                       ^~
make[2]: *** [src/CMakeFiles/kchmviewer.dir/build.make:553: src/CMakeFiles/kchmviewer.dir/qtwebkit/viewwindowmgr.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:203: src/CMakeFiles/kchmviewer.dir/all] Error 2
make: *** [Makefile:152: all] Error 2
```

**Comment by arzam16** [comment]
Turns out one import was missing, adding it has fixed the error. Patch: https://gist.github.com/arzam16/2810542a937edcb819e0c0f7c4616e48

**Comment by u-235** [comment]
Hi, @arzam16!

Thanks for your interest in the project and for the patch in the ef3df18 commit.

**Comment by u-235** [comment]
Fixed in  [v8.1-rc2](https://github.com/u-235/kchmviewer/releases/tag/v8.1-rc2)
